### ISSUE TITLE
Key Items missing in action after Yagudo Torch / before Davoi Crest quest

### ISSUE THREAD

**Issue by Hatberg** [EdenServer/community#699]
Key Items are missing in action, both temporary and permanent.

My last actions were (while making progress on Magicite Mission Rank 5):

- Yesterday night, server r0-ed during cutscene with Muckvix. Decided to call it a night.
- Today logged into server with character Hatburg.
- Talked to Muckvix in Lower Jeuno to obtain cutscene and Yagudo Torch, received Key Item.
- Purchased Couerl Meat from Auction House.
- Zoned from Lower Jeuno to Upper Jeuno.
- Tried to start quest Crest of Davoi from Baudin to obtain key item, got text reply "Baudin : I wish I knew what to do... If you ever find a cure for my sister, please let me know."
- Tried to trade Couerl Meat to Baudin, no response.
- Did not receive Key Item. Tried to check though Menu > Key Items > Permanent Key Items and noticed all KI's were missing.
- Tried talking to Chocobo Stable NPC in Upper Jeuno on a WHM37, but would not allow me to depart.

[https://i.imgur.com/vWwjtBZ.jpg](https://i.imgur.com/vWwjtBZ.jpg)
[https://i.imgur.com/Veo7uQP.jpg](https://i.imgur.com/Veo7uQP.jpg)

**Comment by Hatberg** [comment]
FYI: Zoning, Logging Out and Jobchange have not resolved the issue.

**Comment by Lonewulf724** [comment]
Im in the same boat @Hatberg 

After the patch I was on SMN8 and pt invited me to go do ZM4. I had my key items then because I linked (Sacrificial Chamber Key) in chat.
I warped back to Jeuno, went to MH to change jobs to BLM75.
Went to go to kazham but the npc wouldnt let me through to the airship, this is when I checked my KIs and noticed I didnt have any of them like you. I also noticed I dont have any of my maps for any areas.
